jax/_src/numpy/lax_numpy.py

```diff
@@ -2347,6 +2347,31 @@ def _pad_edge(array, pad_width):
   return array
 
 
+def _pad_linear_ramp(array, pad_width, end_values):
+  for axis in range(ndim(array)):
+    edges = (lax.slice_in_dim(array, 0, 1, axis=axis),
+             lax.slice_in_dim(array, -1, None, axis=axis))
+    ramp_before, ramp_after = (
+        linspace(
+            start=end_value,
+            stop=edge.squeeze(axis), # Dimension is replaced by linspace
+            num=width,
+            endpoint=False,
+            dtype=array.dtype,
+            axis=axis
+        )
+        for end_value, edge, width in zip(
+            end_values[axis], edges, pad_width[axis]
+        )
+    )
```

Comment:
Slight preference here for just writing two linspace operations; reading over this, it took me far longer to understand what the list comprehension was doing than it would have if the two statements were just written out.

Reply:
Thanks for your review. I agree it was hard to understand.
I converted it into two statements. PTAL :)